shared/desktop/yarn-helper/font.js

```diff
@@ -236,6 +236,14 @@ const setFontMetrics = () => {
    * $3: descent value
    */
   const kbTtf = path.resolve(paths.fonts, 'kb.ttf')
+  // Nav icons need to be shifted more because the grid size is 24.
+  // Without shifting to -(64 + 21) the nav icons will be aligned on
+  // a half pixel which will cause blurriness.
+  const icon24 = getSvgNames(true)
+    .filter(({size}) => size === '24')
+    .map(({filePath}) => `'${filePath.replace('.svg', '')}'`)
+  const icon24First = icon24[0]
+  const icon24Last = icon24[icon24.length - 1]
```

Comment:
The discovery here was that the larger icons `kb-iconfont-nav-*-.svg` needed to move moved down an additional amount relative to the other icons.

Comparing the original icon-font to this one, the alignment of all glyphs match exactly now.